src/main/java/training/endava/playground/generics/types/LimitedContainer.java

```diff
@@ -0,0 +1,27 @@
+package training.endava.playground.generics.types;
+
+import java.util.LinkedList;
+import java.util.List;
+
+public class LimitedContainer<T, N> {
+    //I'm not sure what this limited container is supposed to do
+    private LinkedList<Box<T>> container;
+    private N limit;
+
+    public LimitedContainer(List<Box<T>> container, N limit){
+        if (container.size() > (Integer) limit)
```

Comment:
I don't think that's the way of comparing the filed of type T extends Number with an int. what happens if T is Double?
maybe smth like this: https://stackoverflow.com/questions/2683202/comparing-the-values-of-two-generic-numbers